### ISSUE TITLE
brew tap command doesn't find any formulae

### ISSUE THREAD

**Issue by horo42** [mkchoi212/fac#9]
The fac.rb file needs to be in a Formula directory at the root of the repository.

After I ran the brew tap command, cd'd into `/usr/local/Homebrew/Library/Taps/mkchoi212/homebrew-fac` and `mv assets Formula` the `brew install fac` command found the formula and installed it successfully.

**Comment by mkchoi212** [comment]
Please look at https://github.com/mkchoi212/fac/blob/master/.goreleaser.yml#L17

I believe the default is `Formula` if no custom `folder:` is provided but in this case, I believe having `fac.rb` in `assets` is fine.

Just to make sure, are you saying that once you run `brew tap mkchoi212/fac https://github.com/mkchoi212/fac.git`, homebrew is not recognizing the formula?

**Comment by horo42** [comment]
Yes.

```
~ ❯ brew tap mkchoi212/fac https://github.com/mkchoi212/fac.git
==> Tapping mkchoi212/fac
Cloning into '/usr/local/Homebrew/Library/Taps/mkchoi212/homebrew-fac'...
remote: Counting objects: 323, done.
remote: Compressing objects: 100% (286/286), done.
remote: Total 323 (delta 26), reused 310 (delta 25), pack-reused 0
Receiving objects: 100% (323/323), 589.39 KiB | 599.00 KiB/s, done.
Resolving deltas: 100% (26/26), done.
Tapped 0 formulae (321 files, 2.5MB)
```

**Comment by mkchoi212** [comment]
@horo42 Just fixed it! Thank you for letting me know!  2de3d32
